content/guides/userguides/terra-export.md

```diff
@@ -0,0 +1,126 @@
+---
+path: "/guides/userguides/consumer-vignettes/export-to-terra"
+date: "2019-09-17"
+title: "Exporting HCA Data to Terra"
+subTitle: ""
+---
+
+Exporting search results from the HCA Data Explorer to Terra
+============================================================
+
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
+In particular, you'll want to be acquainted with the content in these
+tutorials:
+
+-   [Accessing HCA Data and Metadata][quick-start-guide]
+
+You'll want to use Google Chrome for this tutorial; Terra has poorer support
+for other browsers.
+
+Step one: finding BAM files with the HCA Data Explorer
+------------------------------------------------------
+
+You can use the [HCA Data Explorer][explorer] to find data to export to Terra.
+The Data Explorer lists projects with data available for download from the Data
+Store and lets you filter the data for a number of attributes.
+
+Using the Data Explorer, select some data you're interested in. Choose anything
+that looks interesting - we'll be running a really simple workflow that
+generates MD5 checksums of files, so the type of data doesn't matter too much.
+When you've found what you're looking for, click on the big blue *Export
+Selected Data* button at the top right of the page. You'll see something like
+this:
+
+![The *Export Selected Data* button](_images/terra-export_button.png)
+
+Click on the *Export to Terra* button. You'll then see a page like this where
+you can select what kind of data to export:
+
+![Page for choosing data to export](_images/terra-choose_files.png)
+
+Again, choose whatever looks interesting.
+
+When you click the *Request Export* button, the Data Explorer will process your
```

Comment:
I think we should have some information on how to get a terra login even if it's just a line or two and a link to terra docs.

Reply:
Added to the "Overview of Terra" section